### PR TITLE
Add Common Insurances. Use Env Vars. Update Readme

### PR DESCRIPTION
**Pull request by lacostenycoder**
As per conversation on #42 I updated `spec/support/client.rb` to use environment variables.
Hopefully this will make it a bit easier for others to contribute.

### CODE REVIEW

File: spec/lib/endpoints/insurance_packages_spec.rb [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
     it 'returns instance of InsutancePackageCollection' do
       VCR.use_cassette('top_insurance_packages') do
         expect(client.top_insurance_packages(attributes))
           .to be_an_instance_of AthenaHealth::InsurancePackageCollection
       end
     end
   end
+
+    describe '#common_insurance_packages' do
+      it 'returns instance of InsutancePackageCollection' do
```
Comment (panozzaj) [inline spec/lib/endpoints/insurance_packages_spec.rb]: Slight typo here, should say: `InsurancePackageCollection` (realize it was a copy-paste from above.)

Comment (MatUrbanski) [inline spec/lib/endpoints/insurance_packages_spec.rb]: :+1:

**Comment by lacostenycoder** [comment]
@panozzaj  In my case I just exported the ENV VARS, then ran the tests.  I think you could also use dotenv but not sure if you want to add the dependency in athena_health.gemspec

```
export ATHENA_TEST_API_KEY=<your key>
export ATHENA_TEST_API_SECRET=<your secret>
export ATHENA_TEST_ACCESS_TOKEN=<your token>
```

**Comment by MatUrbanski** [comment]
@lacostenycoder solution is totally fine. Merging.

**Comment by panozzaj** [comment]
Exporting is a good solution, thanks for the update!

**Pull request merged**